substrate/frame/utility/src/lib.rs

```diff
@@ -454,6 +454,36 @@ pub mod pallet {
 			let res = call.dispatch_bypass_filter(frame_system::RawOrigin::Root.into());
 			res.map(|_| ()).map_err(|e| e.error)
 		}
+
+		/// Dispatches a function call with a provided origin.
+		/// Almost the same as [`Pallet::dispatch_as`] but check the result of the call.
+		///
+		/// The dispatch origin for this call must be _Root_.
+		///
+		/// ## Complexity
+		/// - O(1).
+		#[pallet::call_index(6)]
+		#[pallet::weight({
+			let dispatch_info = call.get_dispatch_info();
+			(
+				T::WeightInfo::dispatch_as_checked()
+					.saturating_add(dispatch_info.call_weight),
+				dispatch_info.class,
+			)
+		})]
+		pub fn dispatch_as_checked(
```

Comment:
```suggestion
		pub fn dispatch_as_fallible(
```

Maybe that sounds better?